src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -529,58 +533,36 @@ public boolean verifyNewRankOptionsQuestionFormIsDisplayed() {
         WebElement contribForm = browser.driver.findElement(By.id("rankOptionsForm"));
         return contribForm.isDisplayed() && addNewQuestionButton.isDisplayed();
     }
+    
     public boolean verifyNewRankRecipientsQuestionFormIsDisplayed() {
         WebElement contribForm = browser.driver.findElement(By.id("rankRecipientsForm"));
         return contribForm.isDisplayed() && addNewQuestionButton.isDisplayed();
     }
-    
-    public boolean verifyEndDatesBeforeTodayAreDisabled() {
-        Calendar cal = Calendar.getInstance();
-        
-        // if today is the 1st day of the month, the datepicker cannot navigate to the previous month (yesterday) 
-        if (cal.get(Calendar.DATE) != 1) {
-            cal.add(Calendar.DATE, -1);       
 
-            // get month names
-            DateFormatSymbols dfs = new DateFormatSymbols();
-            String[] months = dfs.getMonths();        
+    public boolean verifyDatesOfPreviousCurrentAndNextMonthAreEnabled() {
```

Comment:
Although the other methods in this class are named <code>verify...</code> . It will be better if you rename this to <code>is...</code>. We've had cases where the return value of <code>verify...</code> were ignored, since <code>verify...</code> sounds it would perform the asserts in it.